packages/extension-chakra-store-locator/package.json

```diff
@@ -80,6 +38,20 @@
     "react-hook-form": "^7.43.9",
     "react-intl": "^5.25.1"
   },
+  "peerDependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@loadable/component": "^5.15.3",
+    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
```

Comment:
nit: Would be better to start with the latest preview version released to NPM? i.e. `^3.2.0-extensibility-preview.3`

Reply:
This was don intentionally. Lerna doesn't know how to account for peer dependencies that are other bootstrapped projects in the mono-repo. As a result npm will try check to ensure this peer dep is on npm, which it is not. This is why we are setting this to the oldest value. That being said, it could be set to `.3` but there is no real benefit. 

Once we go GA we'll be updating these peer deps to point at non-extensibility releases.